ssl/ssl_cert.c

```diff
@@ -488,63 +488,99 @@ STACK_OF(X509_NAME) *SSL_dup_CA_list(STACK_OF(X509_NAME) *sk)
     return (ret);
 }
 
-void SSL_set_client_CA_list(SSL *s, STACK_OF(X509_NAME) *name_list)
+void SSL_set0_CA_list(SSL *s, STACK_OF(X509_NAME) *name_list)
```

Comment:
Well it's little more than an accessor which just sets the internal field to the supplied value so it can't fail.